src/orquestra/sdk/_ray/_dag.py

```diff
@@ -174,6 +174,8 @@ def _make_ray_dag_node(
     ray_kwargs: t.Mapping[str, t.Any],
     pos_unpack_specs: t.Sequence[PosArgUnpackSpec],
     kw_unpack_specs: t.Sequence[KwArgUnpackSpec],
+    pos_deserialize_specs: t.Sequence[int],
+    kw_deserialize_specs: t.Sequence[str],
```

Comment:
**non-blocking**: `pos_deserialize_specs` is confusing to me because it's very similar to `pos_unpack_specs` but it works differently.
* `pos_arg_indices_to_deserialize`?
* `pos_deserialize_specs: t.Sequence[PosDeserializeSpec]` with a separate `PosDeserializeSpec` data structure similar to `PosArgUnpackSpec`?

Reply:
I think creating separate type would be an overkill here - PosArgUpackSpec contains more info, we got just simple type, int or str. But will apply name change and doc it to be more explicit